$NetBSD: patch-ab,v 1.9 2010/04/24 21:27:36 asau Exp $

Adds missing -lsocket needed on solaris for socket(), bind(), and friends.
Unreverses order of CPPFLAGS.

--- configure.in.orig	2009-11-10 21:02:05.000000000 +0300
+++ configure.in	2010-04-25 01:12:31.000000000 +0400
@@ -668,6 +668,7 @@
 
 if test "`uname`" = "SunOS" -o "`uname -sr`" = "HP-UX B.11.00"; then
   dnl ...for Solaris
+  AC_CHECK_LIB([socket], [socket])
   AC_CHECK_LIB([nsl], [xdr_int])
 fi
 
@@ -2172,9 +2173,9 @@
         d="-I${ROOT}/${d}"
         ;;
     esac
-    TEMP_CPPFLAGS="$d $TEMP_CPPFLAGS"
+    TEMP_CPPFLAGS="$TEMP_CPPFLAGS $d"
   done
-  CPPFLAGS=$TEMP_CPPFLAGS
+  CPPFLAGS="$TEMP_CPPFLAGS"
 fi
 
 dnl ----------------------------------------------------------------------
